WorkDays/WorkDays.cs

```diff
@@ -10,32 +10,10 @@ public class WorkDays
         /// Вывести количество календарных дней, которое потребуется для её выполнения.
         /// </summary>
         public int DaysCounter(DateTime current, int workDays)
-        {
-            int firstWeekWorkDays;
+            => (workDays <= 5) ? (((int)current.DayOfWeek + workDays - 1 > 5) ? (5 - (int)current.DayOfWeek + 1) + 2 + (workDays - (5 - (int)current.DayOfWeek + 1)) : workDays)
```

Comment:
(int)current.DayOfWeek - can be extracted to be shortrer

Reply:
done